src/main/java/com/example/b2b_opportunities/Dto/Request/PositionRequestDto.java

```diff
@@ -53,4 +54,6 @@ public class PositionRequestDto {
 
     @NotBlank
     private String description;
+
+    private PositionStatus status;
```

Comment:
When a new Position is added the status will be set automatically to 'Opened', so no need for this